nbgrader/exchange/exchange.py

```diff
@@ -125,3 +125,34 @@ def start(self):
         self.init_src()
         self.init_dest()
         self.copy_files()
+
+    def get_current_user_courses(self):
+        """Check if student is enrolled in course"""
+        from tornado.httpclient import HTTPClient, HTTPRequest
+
+        if os.getenv('JUPYTERHUB_API_TOKEN'):
+            api_token = os.environ['JUPYTERHUB_API_TOKEN']
+        else:
+            self.exit("JUPYTERHUB_API_TOKEN env is required to run the exchange features of nbgrader.")
+        hub_api_url = os.environ.get('JUPYTERHUB_API_URL') or 'http://127.0.0.1:8081/hub/api'
+        if os.getenv('JUPYTERHUB_USER'):
+            user = os.environ['JUPYTERHUB_USER']
+        else:
+            self.exit("JUPYTERHUB_USER env is required to run the exchange features of nbgrader.")
+        import json
+        auth_header = {
+                'Authorization': 'token %s' % api_token
+            }
+        client = HTTPClient()
+        req = HTTPRequest(url=hub_api_url + '/users/%s' % user,
+            method='GET',
+            headers=auth_header,
+        )
+
+        resp = client.fetch(req)
+        groups = json.loads(resp.body.decode('utf8', 'replace'))['groups']
+        courses = set()
+        for group in groups:
+            if group.startswith('nbgrader-') or group.startswith('formgrade-'):
```

Comment:
So we're basically making a strong assumption here that all groups that restrict access to nbgrader courses will be prefixed with `nbgrader-` or `formgrade-`. That's fine, I just am noting this so it can be added to the documentation here.

Reply:
ok, I added a list of items to add to the docs in the top post